### PR TITLE
using super in robot and commands

### PR DESCRIPTION
**Pull request by rfeistenauer**
This should fix Issue #115

### CODE REVIEW

File: pilz_robot_programming/src/pilz_robot_programming/robot.py [hunk 1]
```diff
@@ -105,7 +105,8 @@ class Robot(object):
     _PID_STRING = "pid"
     _PROCESS_CREATE_TIME_STRING = "create_time"
 
-    def __init__(self, version=None):
+    def __init__(self, version=None, **kwargs):
+        super(Robot, self).__init__()
```
Comment (rfeistenauer) [inline pilz_robot_programming/src/pilz_robot_programming/robot.py]: probalby it should even be 
```
def __init__(version=None, *args, **kwargs)
    super(Robot, self).__init__(*args, **kwargs)
```

**Comment by agutenkunst** [comment]
There are checks failing. Could you check on this?

**Comment by agutenkunst** [comment]
There is a typo in the first commit and maybe add "Closes" for automatic issue closing in your commit.

**Comment by rfeistenauer** [comment]
```
class Ptp(_BaseCmd):
    """
    ...
    """
    def __init__(self, acc_scale=None, vel_scale=None, *args, **kwargs):      
        acc_scale_final = acc_scale if acc_scale is not None else Ptp._calc_acc_scale(vel_scale)        
        super(Ptp, self).__init__(acc_scale=acc_scale_final, vel_scale=vel_scale, *args, **kwargs)
```
Depending on whether you want it for documentation reasons or not, you could even throw out all kwargs out of the command classes Ptp, Lin and Circ. They will still show on code completion hint since super takes care of that, but the doc above the classes may look off place.

**Review by ct2034 — CHANGES_REQUESTED** [review]
I think it would make sense to also remove the explicit params in the inheriting classes (when calling inti with super)

**Review by ct2034 — APPROVED** [review]

**Comment by ct2034** [comment]
@dbakovic @agutenkunst @hslusarek @martiniil 
Do we want this? I think it is a lot more clean

**Review by martiniil — COMMENTED** [review]
I don't see why someone would inherit from the Robot class multiple times. For single inheritance super() in the base class has no effect, right?
 
@rfeistenauer What are your thoughts on that?

**Comment by martiniil** [comment]
I got it.

**Comment by rfeistenauer** [comment]
just for documentation purpose: https://www.youtube.com/watch?v=EiOglTERPEo answeres this question

**Pull request merged**